.env

```
@@ -1 +1,5 @@
 API_URL=https://api.nhlstats.org
+
+CONVEX_DEPLOYMENT=dev:ideal-peccary-58 # team: maxime-pelletier-5d389, project: nhlstats
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)